app/scripts/modules/kubernetes/pipeline/stages/runJob/runJobStage.js

```diff
@@ -52,4 +83,38 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.kubernetes.runJob
     this.onChange = (changes) => {
       this.stage.container.imageDescription.registry = changes.registry;
     };
+
+    this.triggerImages = () => {
+
+      if (this.pipeline.triggers.length <= 0) {
+        return [];
+      }
+
+      return this.pipeline.triggers.filter((trigger) => {
+        return trigger.type === 'docker' && trigger.enabled === true;
+      }).map((trigger) => {
+        return buildImageDescriptor(trigger);
+      });
+    };
+
+    this.hasDockerPipelineTriggers = () => {
+      return this.triggerImages().length > 0;
+    };
+
+    this.updateContainerImage = () => {
+      //filter the list of triggers and take the first one that matches
+      const trigger = this.pipeline.triggers.filter((trigger) => {
```

Comment:
@danielpeach I took a different approach with this function. Instead of trying to glean all info from the drop down  value, I use that to target the specific trigger and pull the relevant information from that. It uses the first one it finds that matches.